Makefile

```diff
@@ -14,6 +14,10 @@ NON_MATCHING ?= 0
 ORIG_COMPILER ?= 0
 # If COMPILER is "gcc", compile with GCC instead of IDO.
 COMPILER ?= ido
+# Target game version. Currently the following versions are supported:
+#   gc-eu-mq       GameCube Europe/PAL Master Quest
```

Comment:
Id remove the non debug comment until we actually support it

Reply:
Yeah, I guess I meant "supported by the Makefile" but it was confusing. I clarified that gc-eu-mq is a WIP